docs/examples/modular.rst

```diff
@@ -4,10 +4,16 @@
 Modular Contracts
 *****************
 
-A modular approach to building your contracts helps you prevent overflow risks
-and unexpected tokens. In the example below, the contract uses the ``move`` method
+A modular approach to building your contracts helps you reduce the complexity.
+If you specify and control the behaviour or each module in isolation, the
+interactions you have to consider are only those between the specifications
+and not every other moving part of the contract.
+In the example below, the contract uses the ``move`` method
 of the ``Balances`` :ref:`library <libraries>` to check that balances sent between
-addresses match what you expect.
+addresses match what you expect. In this way, the ``Balances`` library
+provides an isolated component that properly tracks balances of accounts.
+It is easy to verify that it never produces negative balances or overflows
```

Comment:
`It is easy to verify that the ``Balances`` library never produces negative balances...`? (replaced it by the Balances library)